schema/gnosis_safe/view_safes.sql

```diff
@@ -0,0 +1,66 @@
+create materialized view gnosis_safe.view_safes as
+    with safe_setups as (
+        select call_tx_hash as tx_hash, _threshold as threshold, LENGTH(REPLACE(text(_owners),'\\x','~'))-LENGTH(REPLACE(text(_owners),'\\x','')) as owners
```

Comment:
Are you calculating the length of the `_owners` array here? I wonder if it would be better to just use `array_length(_owners, 1)`, unless there's a reason not to that I can't spot?

Reply:
nope, no reasons, just changed it